public/app/percona/backup/components/BackupInventory/BackupInventory.tsx

```diff
@@ -304,8 +327,15 @@ export const BackupInventory: FC = () => {
               onDelete={handleDelete}
               initialForceValue={true}
               loading={deletePending}
-              showForce
-            />
+              showForce={!!selectedBackup && backupLocationMap.current[selectedBackup.id]?.type !== LocationType.CLIENT}
+            >
+              {!!selectedBackup && backupLocationMap.current[selectedBackup.id]?.type === LocationType.CLIENT && (
+                <Alert title="">
+                  For security reasons, we won&apos;t delete this backup from the filesystem. Please make sure to
```

Comment:
Can you move this text to a separate file?

Reply:
sure!